.github/workflows/deploy-prod.yaml

```diff
@@ -6,6 +6,21 @@ on:
     - "v*.*.*"
 
 jobs:
+  verify-tag:
+    runs-on: ubuntu-20.04
+    outputs:
+        result: ${{ steps.verify-tag-id.outputs.result }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - id: verify-tag-id
+      name: Verify version tag
+      run: |
+          export VERSION_TAG=$(echo $GITHUB_REF | awk -F'/' '{ print $NF }')
+          OUTPUT=`bin/verify-tag.sh "${VERSION_TAG}"`
+          echo "::set-output name=result::$OUTPUT"
```

Comment:
no need to output here as you are not using this as an input https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs